src/components/pages/ParentHome/ParentCalendar.js

```diff
@@ -1,61 +1,185 @@
 import React, { useState } from 'react';
 import '../../../styles/ParentStyles/index.less';
 import 'antd/dist/antd.css';
-
 import { Calendar, Badge, Modal, Button } from 'antd';
 import { InfoCircleOutlined } from '@ant-design/icons';
-import { connect } from 'react-redux';
+import { dateConverter } from '../../common/dateHelpers';
+import { timeConverter } from '../../common/timeHelpers';
+
+const dummyAvailableCourses = [
+  {
+    session_id: 1,
+    course_id: 1,
+    instructor_id: 1,
+    instructor_name: 'Test003',
+    instructor_rating: 2,
+    size: 15,
+    subject: 'CS101',
+    description: 'Computer Science fundamentals',
+    prereqs: [],
+    start_date: '2021-12-10T07:00:00.000Z',
+    end_date: '2022-10-10T07:00:00.000Z',
+    start_time: '17:00:00',
+    end_time: '18:00:00',
+    location: 'https://zoom.us/my/john123',
+  },
+  {
+    session_id: 3,
+    course_id: 5,
+    instructor_id: 10,
+    instructor_name: 'Mark',
+    instructor_rating: 5,
+    size: 20,
+    subject: 'Fundamental Python',
+    description: 'Computer Science fundamentals',
+    prereqs: ['JavaScript', 'HTML', 'CSS'],
+    start_date: '2021-12-10T07:00:00.000Z',
+    end_date: '2022-10-10T07:00:00.000Z',
+    start_time: '17:00:00',
+    end_time: '18:00:00',
+    location: 'https://zoom.us/my/john123',
+  },
+  {
+    session_id: 2,
+    course_id: 3,
+    instructor_id: 2,
+    instructor_name: 'Test006',
+    instructor_rating: 4,
+    size: 16,
+    subject: 'JavaScriptB',
+    description: 'Intermediate JavaScript.',
+    prereqs: ['JavaScriptA'],
+    start_date: '2021-12-05T07:00:00.000Z',
+    end_date: '2022-10-11T07:00:00.000Z',
+    start_time: '15:00:00',
+    end_time: '16:00:00',
+    location: 'https://zoom.us/my/john321',
+  },
+];
+
+function displayedItem(value, current, item) {
+  return {
+    type: current === value ? 'warning' : 'success',
+    session_id: item.session_id,
+    course_id: item.course_id,
+    instructor_id: item.instructor_id,
+    instructor_name: item.instructor_name,
+    instructor_rating: item.instructor_rating,
+    size: item.size,
+    subject: item.subject,
+    description: item.description,
+    prereqs: item.prereqs,
+    start_date: item.start_date,
+    end_date: item.end_date,
+    start_time: item.start_time,
+    end_time: item.end_time,
+    location: item.location,
+  };
+}
 
-function ParentCalendar(props) {
-  const { schedule } = props;
-  const [course, setCourse] = useState('');
+function Sessions(props) {
+  const { listData, activeModal, showModal, handleCancel, handleClick } = props;
+  return (
+    <ul className="ulcell">
+      {listData.map(item => {
```

Comment:
What you did here is incredible, and I didn't even know we could do that.
your code is dry and looks great